packages/cli/src/lib/cmds/play.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {
+  Config, GradleWrapper, JdkHelper, AndroidSdkTools, ConsoleLog, Log, GooglePlay, TwaManifest,
+} from '@bubblewrap/core';
+import {Track} from '@bubblewrap/core/dist/lib/GooglePlay';
+import * as fs from 'fs';
+import * as path from 'path';
+import {ParsedArgs} from 'minimist';
+import {TWA_MANIFEST_FILE_NAME} from '../constants';
+import {Prompt, InquirerPrompt} from '../Prompt';
+/**
+  * The Play class is the class that is used to communicate with the Google Play Store.
+  */
+class Play {
+  constructor(
+    private args: ParsedArgs,
```

Comment:
We haven't been very consistent about this, but we may want to introduce a `PlayArgs` that helps us with type checking. We [have one for init](https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/cli/src/lib/cmds/init.ts#L29-L34) and we [cast `ParsedArgs` to that type](https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/cli/src/lib/Cli.ts#L70) when invoking the command.

Reply:
Think I got this one done.